main.go

```diff
@@ -0,0 +1,94 @@
+/**
+ *Author: Stefan
+ *Date: 11/24/2019
+ *Last changes: 11/26/2019 10.23
+ *Task: Implement sort.Interface interface for People type.
+ *				Sort people in order of increasing age.
+ *				If two people have the same age - sort them by name.
+ *				Add test cases.
+ */
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"sort"
+	"time"
+)
+
+//Person struct
+type Person struct {
+	firstName string
+	lastName  string
+	birthDay  time.Time
+}
+
+//People struct
+type People []Person
+
+//Len method
+func (p People) Len() int {
+	return len(p)
+}
+
+//Less method
+func (p People) Less(i, j int) bool {
+
+	if p[i].birthDay.Sub(p[j].birthDay) > 0 {
+		return true
+	} else if p[i].birthDay.Sub(p[j].birthDay) < 0 {
+		return false
+	} else if p[i].firstName < p[j].firstName {
```

Comment:
this shouldn't be `else if`, just a new `if`. This func seems to be sophisticated, but still it's OK.

Reply:
Thanks! Done:)